sdk/eventgrid/Azure.Messaging.EventGrid.SystemEvents/src/Generated/MediaLiveEventIncomingDataChunkDroppedEventData.cs

```diff
@@ -75,8 +73,6 @@ internal MediaLiveEventIncomingDataChunkDroppedEventData(string timestamp, strin
         public string TrackType { get; }
         /// <summary> Gets the bitrate of the track. </summary>
         public long? Bitrate { get; }
-        /// <summary> Gets the timescale of the Timestamp. </summary>
-        public string Timescale { get; }
```

Comment:
this property is removed because this property has a `@visibility` with no values.
![image](https://github.com/user-attachments/assets/4cd36d0f-3cdc-49d4-8eca-06caa8c596ac)
I am not quite sure whether this is by design or a regression in typespec new version or tcgc's new version yet.

Reply:
This entire service was removed in a later commit so this is expected.